odoo-shippable/scripts/library.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+git_clone_copy(){
```

Comment:
Thanks for clarify

Could you send me the `docker pull` command to download it or I should wait the not-WIP state?